charts/prometheus/templates/alertmanager/deploy.yaml

```diff
@@ -155,9 +159,9 @@ spec:
       dnsConfig:
 {{ toYaml . | indent 8 }}
     {{- end }}
-    {{- if .Values.alertmanager.securityContext }}
+    {{- if .Values.alertmanager.securityContext.enabled }}
       securityContext:
-{{ toYaml .Values.alertmanager.securityContext | indent 8 }}
+{{ omit .Values.alertmanager.securityContext "enabled" | toYaml | indent 8 }}
     {{- end }}
```

Comment:
Fix indent.

```suggestion
      {{- if .Values.alertmanager.securityContext.enabled }}
      securityContext:
        {{- omit .Values.alertmanager.securityContext "enabled" | toYaml | nindent 8 }}
      {{- end }}
```

Reply:
@zanhsieh 
Indentation is correct here and aligned with indentation for other values:
```
...
{{ toYaml .Values.imagePullSecrets | indent 8 }}
    {{- end }}
    {{- if .Values.alertmanager.nodeSelector }}
      nodeSelector:
{{ toYaml .Values.alertmanager.nodeSelector | indent 8 }}
    {{- end }}
    {{- with .Values.alertmanager.dnsConfig }}
      dnsConfig:
{{ toYaml . | indent 8 }}
    {{- end }}
    {{- if .Values.alertmanager.securityContext.enabled }}
      securityContext:
{{ omit .Values.alertmanager.securityContext "enabled" | toYaml | indent 8 }}
    {{- end }}
    {{- if .Values.alertmanager.tolerations }}
      tolerations:
{{ toYaml .Values.alertmanager.tolerations | indent 8 }}
    {{- end }}
    {{- if .Values.alertmanager.affinity }}
      affinity:
{{ toYaml .Values.alertmanager.affinity | indent 8 }}
...
```
Also linting is failed with your suggested changes:
```
==> Linting .
[ERROR] templates/alertmanager/deploy.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 74: did not find expected key

Error: 1 chart(s) linted, 1 chart(s) failed
```